package node;

message LoginRequest {
  required string originip = 1;
  required uint32 routecount = 2;
  required uint64 account = 3;
  optional uint32 version = 4;
  optional string remoteip = 5;
}

